lite/examples/posenet/android/posenet/src/main/java/org/tensorflow/lite/examples/posenet/lib/Posenet.kt

```diff
@@ -124,16 +124,18 @@ class Posenet(
     inputBuffer.order(ByteOrder.nativeOrder())
     inputBuffer.rewind()
 
+    val pixels = IntArray(bitmap.height * bitmap.width)
+    bitmap.getPixels(pixels, 0, bitmap.width, 0, 0, bitmap.width, bitmap.height)
+
     val mean = 128.0f
     val std = 128.0f
-    for (row in 0 until bitmap.height) {
-      for (col in 0 until bitmap.width) {
-        val pixelValue = bitmap.getPixel(col, row)
-        inputBuffer.putFloat(((pixelValue shr 16 and 0xFF) - mean) / std)
-        inputBuffer.putFloat(((pixelValue shr 8 and 0xFF) - mean) / std)
-        inputBuffer.putFloat(((pixelValue and 0xFF) - mean) / std)
-      }
+    for (pixelValue in pixels)
```

Comment:
Nit: please put the opening paren on the for loop line

```
 for (pixelValue in pixels) {
```

Reply:
I have made the requested changes